fence/agents/scsi/fence_scsi.py

```diff
@@ -182,7 +182,14 @@ def get_cluster_id(options):
 	cmd = options["--corosync-cmap-path"] + " totem.cluster_name"
 
 	match = re.search(r"\(str\) = (\S+)\n", run_cmd(options, cmd)["out"])
-	return hashlib.md5(match.group(1)).hexdigest() if match else fail_usage("Failed: cannot get cluster name")
+
+	if not match:
+		fail_usage("Failed: cannot get cluster name")
+
+	try:
+		return hashlib.md5(match.group(1), usedforsecurity=False).hexdigest()
+	except TypeError:
+		return hashlib.md5(match.group(1)).hexdigest()
```

Comment:
My Fedora has it and I had not tested it on the latest one, so that's a right point.

No, I said that RHEL/CentOS/Fedora is a huge majority of our users not that they are running FIPS. And for RHEL/CentOS it makes sense to have usedforsecurity=False.